.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+**This pull request makes the following changes:**
+* Fixes issue #issueNumberHere 
```

Comment:
@samuko-things Change this line to:
- Fixes issue code-collabo/repo-name-here#issueNumberHere

Reply:
i have made the changes